android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -485,6 +496,24 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
         }
     }
 
+    fun cacheCleanup(envoyService: String) {
+        // clean up http cache dir
+        val cacheDir = File(applicationContext.cacheDir, envoyService + "-cache")
+        if (cacheDir.exists()) {
```

Comment:
cacheDir is guaranteed to be flat?

Reply:
i think the cache directory we create may have subdirectories if that's what you mean